developer-docs-site/docs/tutorials/validator-node/connect-to-testnet.md

```diff
@@ -12,17 +12,20 @@ Only do this if you got confirmation email from Aptos team for your eligibility.
 
 - Stop your node and remove the data directory.
 - Download the `genesis.blob` and `waypoint.txt` file published by Aptos Labs team.
+- Pull the latest changes on `testnet` branch, make sure you're at commit `f032dc15`
 - Restarting the node
 
 ## Using Docker
 
 - Stop your node and remove the data volumes, `docker-compose down --volumes`
 - Download the `genesis.blob` and `waypoint.txt` file published by Aptos Labs team.
+- Update your docker image to use tag `testnet_f032dc15`
```

Comment:
maybe include the link to the page on dockerhub? they can get the sha to pin the image too from here https://hub.docker.com/layers/validator/aptoslab/validator/testnet_f032dc15/images/sha256-322c773dce228afcf17a4d65b12e7a372ab93693ee59566cac6794cf3a26b1d2?context=repo